components/server/src/authorization/relationship-updater.ts

```diff
@@ -33,9 +35,21 @@ export class RelationshipUpdater {
      * @returns
      */
     public async migrate(user: User): Promise<User> {
+        const fgaEnabled = await getExperimentsClientForBackend().getValueAsync("centralizedPermissions", false, {
+            user: {
+                id: user.id,
+            },
+        });
+        if (!fgaEnabled) {
+            return user;
+        }
         if (user.additionalData?.fgaRelationshipsVersion === this.version) {
             return user;
         }
+        log.info({ userId: user.id }, `Updating FGA relationships for user.`, {
+            fromVersion: user?.additionalData?.fgaRelationshipsVersion,
+            toVersion: this.version,
+        });
         // TODO run in distributed lock
         // return this.mutex.using([`fga-migration-${user.id}`], 2000, async () => {
```

Comment:
Should we remove this comment?

Reply:
we should uncomment the code and make it work in our GHA build. I had to disable it because redis is not working properly in GHA.